### ISSUE TITLE
'get barrel' in tutorial elicits a traceback.

### ISSUE THREAD

**Issue by feoh** [evennia/ainneve#39]
I was using a newly created player with no extra privs:

```
Traceback (most recent call last):
  File "/home/feoh/src/personal/evennia/evennia/commands/cmdhandler.py", line 417, in _run_command
    ret = yield cmd.func()
  File "/home/feoh/src/personal/evennia/evennia/contrib/tutorial_world/objects.py", line 1006, in
    func
    self.obj.produce_weapon(self.caller)
  File "/home/feoh/src/personal/evennia/evennia/contrib/tutorial_world/objects.py", line 1063, in
    produce_weapon
    wpn = spawn(WEAPON_PROTOTYPES[prototype], prototype_parents=WEAPON_PROTOTYPES)[0]
  File "/home/feoh/src/personal/evennia/evennia/utils/spawner.py", line 189, in spawn
    for key, val in all_from_module(prototype_module).items() if isinstance(val, dict)))
  File "/home/feoh/src/personal/evennia/evennia/utils/utils.py", line 803, in all_from_module
    mod = mod_import(module)
  File "/home/feoh/src/personal/evennia/evennia/utils/utils.py", line 767, in mod_import
    mod = __import__(module, fromlist=["None"])
ImportError: No module named prototypes

Above traceback is from an untrapped error. Please file a bug report.
```

**Comment by ergodicbreak** [comment]
This bug looks like an effect of the reorganization of Ainneve's world dir, so Evennia is not finding the default prototype module path. Can you confirm this works on vanilla Evennia without using Ainneve?

**Comment by feoh** [comment]
@ergodicbreak That's correct, when I run through the tutorial in Evennia proper (No Ainneve) I don't get the traceback.

**Comment by thinkt4nk** [comment]
+1

**Comment by feend78** [comment]
It seems to me that we probably don't want to encourage people to build the Evennia tutorial world in the first place. A solution for this would be to modify BASE_BATCHPROCESS_PATHS in settings.py to no longer include the default Evennia locations, and specifically only the **world** directory instead.

That's a super easy change, but then we would need to somehow update the description for _Limbo_ to give a customized Ainneve welcome message for admins and/or developers, and possibly a similar build instruction, once we have batch files that will build the Ainneve world. This is the part I'm unsure of. 

I've done a test where I tried adding a new description for _Limbo_ in the **at_initial_setup** hook, but it didn't seem to work. The docs say that hook is run "at the same time as the super user and Limbo are created," but I wonder if it's actually before those things happen... In any case, I'll to do some more research, but if someone more knowledgeable than I has a suggestion for the best way to approach modifying _Limbo_ before the first login, it would be appreciated.

**Comment by feend78** [comment]
I dug a little further into this, and the error is occurring because in ainneve:master, the module that settings.py points to for prototype definitions is not present. This issue could be worked around by creating an empty file **annieve/world/prototypes.py**. I want to hold off on submitting this as a fix, though, because in response to Issue #20, I've already submitted a pull request that modifies the prototype settings, and if accepted, will also fix this issue.

As an aside, I have figured out to update the description for _Limbo_, and will likely be submitting a PR with an Ainneve development-specific room description, just for kicks.

**Comment by ergodicbreak** [comment]
Agreed that we don't really need to support the tutorial world. Will leave this open until the PR from @feend78 fixes.

**Comment by feend78** [comment]
That PR went through a while ago. I think this can be closed.

**Comment by ergodicbreak** [comment]
IIRC @feend78 closed that PR before merging, in order to work on other things first? Looking at our settings.py I don't think the change was merged in any case.

**Comment by ergodicbreak** [comment]
Fixed by #53
